### PR TITLE
fix(card): rename slot content to main

### PR DESCRIPTION
**Pull request by rachelbt**

### CODE REVIEW

File: libs/components/src/lib/card/README.md [hunk 1]
```diff
@@ -104,22 +104,20 @@ The meta slot is for action content in the card header
...
 
-### Content
+### Default
 
 If the content structure is not fitting the desired content, the heading, subheading and text can overide
 
 ```html preview
 <vwc-card >
-<div slot="content">If you need different structure - use this slot</div>
+<div slot="default">If you need different structure - use this slot</div>
```
Comment (yinonov) [inline libs/components/src/lib/card/README.md:120]: default slot just picks any nested content which doesn't have a named slot attribute
```suggestion
<div>If you need different structure - use this slot</div>
```

File: libs/components/src/lib/card/card.template.ts [hunk 2]
```diff
@@ -100,14 +100,14 @@ export const CardTemplate: (
...
-	<vwc-elevation dp=${(x => x.elevation ??  '4')}>
+	<vwc-elevation dp=${(x => x.elevation ?? '4')}>
 		<div class="${getClasses}">
 			<div class="wrapper">
 				<div class="vwc-card-media">
 					<slot name="media"></slot>
 				</div>
 				<div class="content">
-					<slot name="content">
+					<slot name="default">
```
Comment (yinonov) [inline libs/components/src/lib/card/card.template.ts:110]:
```suggestion
					<slot>
```

**Review by yinonov — CHANGES_REQUESTED** [review]

File: libs/components/src/lib/card/README.md [hunk 3]
```diff
@@ -82,7 +82,7 @@ The graphic slot overide the icon property
 The media slot is mainly for images or video content above the card header
 
 ```html preview
-<div style="block-size: 400px; inline-size: 300px;">
+<div style="inline-size: 300px;">
```
Comment (yinonov) [inline libs/components/src/lib/card/README.md:85]: still needed?

File: libs/components/src/lib/card/README.md [hunk 4]
```diff
@@ -82,7 +82,7 @@ The graphic slot overide the icon property
...
 <vwc-card heading="Card with Media" text="here is the card text">
```
Comment (yinonov) [inline libs/components/src/lib/card/README.md:86]: try to keep formatting (indentation) clear for readers

**Review by yinonov — CHANGES_REQUESTED** [review]

File: libs/components/src/lib/card/README.md [hunk 5]
```diff
@@ -103,19 +103,23 @@ footer slot main purpose is for action button
...
   <vwc-button slot="meta" icon="more-vertical-solid" appearance="ghost"></vwc-button>
   <vwc-button slot="footer" icon="arrow-bold-right-line" shape="pill" label="Action" appearance="outlined"></vwc-button>
 </vwc-card>
 ```
 
-### Content
+### Main
 
-If the content structure is not fitting the desired content, the heading, subheading and text can overide
+main slot is for overriding the content structure of the main card area.
```
Comment (yinonov) [inline libs/components/src/lib/card/README.md:114]: maybe elaborate on why. we have our opinionated style which can sometimes require overriding etc'.

**Review by yinonov — CHANGES_REQUESTED** [review]

File: libs/components/src/lib/card/ui.test.ts [hunk 6]
```diff
@@ -7,7 +7,7 @@ import {
 	loadTemplate,
 } from '../../visual-tests/visual-tests-utils.js';
 
-const components = ['card'];
+const components = ['card', 'button', 'layout'];
```
Comment (yinonov) [inline libs/components/src/lib/card/ui.test.ts:10]: where do we use layout?

Comment (yinonov) [reply libs/components/src/lib/card/ui.test.ts:10]: I'm importing layout in the base file now

**Review by yinonov — CHANGES_REQUESTED** [review]

**Review by yinonov — CHANGES_REQUESTED** [review]

File: libs/components/src/lib/card/README.md [hunk 7]
```diff
@@ -103,19 +103,24 @@ footer slot main purpose is for action button
...
 
-If the content structure is not fitting the desired content, the heading, subheading and text can overide
+We offer our opinionated structure for the card.  
+Assign nodes to `main` slot to fully override a card's main template.
 
 ```html preview
-<vwc-card >
-  <div slot="content">If you need different structure - use this slot</div>
+<vwc-card>
+  <vwc-layout slot="main">
```
[the remark this answers is not in the document]
Comment (rachelbt) [reply libs/components/src/lib/card/README.md:119]: why should we add that?

[the remark this answers is not in the document]
Comment (yinonov) [reply libs/components/src/lib/card/README.md:119]: then why are you using layout if it applies no style?

[the remark this answers is not in the document]
Comment (rachelbt) [reply libs/components/src/lib/card/README.md:119]: there was vwc-text margin so I didn't notice this is needed :woman-facepalming

**Review by yinonov — DISMISSED** [review]

**Review by yinonov — APPROVED** [review]

**Pull request merged**